tests/integration/expected_test_results/api_v3_product_write/patch-broken-json-body.json

```diff
@@ -2,7 +2,7 @@
    "errors" : [
       {
          "field" : {
-            "error" : "'null' expected, at character offset 0 (before \"not json\") at /opt/product-opener/lib/ProductOpener/API.pm line 237.\n",
+            "error" : "--ignore--",
```

Comment:
I think we should only ignore the line number and keep the rest of the error message